CompetitionTemplate/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -16,28 +16,24 @@
     // For mapping operator interface buttons to commands
     
     @Inject
-    public void setupCollectorCommands(OperatorInterface oi, IntakeCollectorCommand intakeCommand, EjectCollectorCommand ejectCommand){
-        oi.leftButtons.getifAvailable(1).whileHeld(intakeCommand);
-        oi.leftButtons.getifAvailable(3).whileHeld(ejectCommand);
+    public void setupCollectorCommands(OperatorInterface oi, IntakeCollectorCommand intakeCommand){
+        oi.leftButtons.getifAvailable(5).whileHeld(intakeCommand);
     }
     
     @Inject
     public void setupHoodCommands(OperatorInterface oi, ExtendHoodCommand extendCommand, RetractHoodCommand retractCommand){
-        oi.leftButtons.getifAvailable(2).whileHeld(extendCommand);
-        oi.leftButtons.getifAvailable(4).whileHeld(retractCommand);
+        oi.leftButtons.getifAvailable(/*TODO: D-pad down*/).whileHeld(extendCommand);
```

Comment:
If you integrate from master again, you should get access to the Dpad joystick on the gamepad